linters.py

```diff
@@ -1097,7 +1097,7 @@ def _instance(self) -> Linter:
             self.__module_path.rstrip('.py').replace('/', '_'),
             self.__module_path)
         custom_linter_module = importlib.util.module_from_spec(
-            custom_linter_module_spec)
+            custom_linter_module_spec)  # type: ignore
```

Comment:
```suggestion
            custom_linter_module_spec)
```

siempre hay que evitar el uso de `# type: ignore`. en este caso es porque legalmente puede regresar `None` y hay que tomar ese caso en cuenta!

Reply:
Listo.